mne/decoding/__init__.py

```diff
@@ -10,4 +10,5 @@
 from .time_gen import GeneralizationAcrossTime, TimeDecoding
 from .time_frequency import TimeFrequency
 from .receptive_field import ReceptiveField
+from .time_delaying_ridge import TimeDelayingRidge
```

Comment:
why not `mne.decoding.model` instead of having its own module. That way if we want to add any custom models they can go in there?